CMakeLists.txt

```
@@ -155,7 +155,7 @@ endif()
 
 if(DD4HEP_USE_EDM4HEP)
   find_package(EDM4HEP REQUIRED)
-  find_package(podio 0.16.3 REQUIRED)
+  find_package(podio REQUIRED)
```

What about this approach? [1]
```suggestion
  find_package(podio 0.16.3)
  if(NOT podio_FOUND)
    find_package(podio 1.0 REQUIRED)
  endif()
```
[1]: https://github.com/eic/EDM4eic/pull/84

---

```suggestion
    # we need podio with Frame support (>=0.16.3)
    # podio is "SameMajorVersion" compatible
    find_package(podio 0.16.3)  # this will not find 1.0 and newer
  if(NOT podio_FOUND)
    # we try to find a newer version now
    find_package(podio 1.0 REQUIRED)
  endif()
```